readme.md

```diff
@@ -113,6 +143,13 @@ Default: The `name` field in the package.json closest to where `conf` is importe
 
 You only need to specify this if you don't have a package.json file in your project.
 
+#### projectVersion
+
+Type: `string`<br>
+Default: The `version` field in the package.json closest to where `conf` is imported.
+
+You only need to specify this if you don't have a package.json file in your project.
```

Comment:
You only need to specify this if you don't have a package.json file in your project or if there isn't a version defined within it.

Reply:
Done. I did the same for the name!